docs/letting-go.md

```diff
@@ -0,0 +1,62 @@
+
+## Converting to from Data Hub to IML
+
+> Data Hub is currently "[In Maintenance](https://developer.here.com/documentation/product-lifecycle-policy/customer_notice/index.html)". Interactive map layers (IML) has been identified as the successor to Data Hub. For new users considering Data Hub, HERE recommends that you start with IML on the [HERE platform](https://platform.here.com).
```

Comment:
Why the '>'  for a quote? What do you quote? Source?

Reply:
Copied it from here: https://github.com/heremaps/xyz-documentation/blob/master/docs/api/README.md

Reply:
But your version does not have the "Note" header. That would make more sense.